packages/completer/src/tokens.ts

```diff
@@ -378,6 +383,13 @@ export interface ICompletionProviderManager {
   inlineProviders?: IInlineCompletionProviderInfo[];
 }
 
+export interface ICompleterSelection {
+  /**
+   * The text selected by the completer.
+   */
+  text: string;
```

Comment:
I was thinking about using `insertText` to align with the other interfaces. Not a strong preference though.

```suggestion
  insertText: string;
```

Reply:
No preference from me, I'll make this change